yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -240,80 +240,39 @@ func prepareImportDataStatusTable(streamChanges bool) ([]*tableMigStatusOutputRo
 		}
 	}
 
-	for _, dataFile := range dataFileDescriptor.DataFileList {
-		var totalCount, importedCount int64
-		var err error
-		var perc float64
-		var status string
-		reportProgressInBytes = reportProgressInBytes || dataFile.RowCount == -1
-		totalCount = dataFile.RowCount
-		if reportProgressInBytes {
-			importedCount, err = state.GetImportedByteCount(dataFile.FilePath, dataFile.TableName)
-		} else {
-			if importerRole == FB_DB_IMPORTER_ROLE {
-				importedCount = snapshotRowCount[dataFile.TableName]
-			} else {
-				importedCount, err = state.GetImportedRowCount(dataFile.FilePath, dataFile.TableName)
-			}
-		}
-		if err != nil {
-			return nil, fmt.Errorf("compute imported data size: %w", err)
-		}
-
-		if totalCount != 0 {
-			perc = float64(importedCount) * 100.0 / float64(totalCount)
-		}
-		switch true {
-		case importedCount == totalCount:
-			status = "DONE"
-		case importedCount == 0:
-			status = "NOT_STARTED"
-		case importedCount < totalCount:
-			status = "MIGRATING"
-		}
-		row := &tableMigStatusOutputRow{
-			tableName:          dataFile.TableName,
-			schemaName:         getTargetSchemaName(dataFile.TableName),
-			fileName:           path.Base(dataFile.FilePath),
-			status:             status,
-			totalCount:         totalCount,
-			importedCount:      importedCount,
-			percentageComplete: perc,
-		}
-		if streamChanges {
-			eventCounter, err := state.GetImportedEventsStatsForTable(row.tableName, migrationUUID)
-			if err != nil {
-				return nil, fmt.Errorf("get imported events stats for table %q: %w", row.tableName, err)
-			}
-			if importerRole == FB_DB_IMPORTER_ROLE {
-				schemaNameForQuery := ""
-				tableNameForQuery := row.tableName
-				tableParts := strings.Split(row.tableName, ".")
-				if len(tableParts) == 2 {
-					schemaNameForQuery = tableParts[0]
-					tableNameForQuery = tableParts[1]
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		return nil, fmt.Errorf("get migration status record: %w", err)
+	}
+	if msr.SourceDBConf != nil {
+		source = *msr.SourceDBConf
+	}
+	importTableList := getImportTableList(msr.TableListExportedFromSource)
+	if streamChanges {
+			for _, tableName := range importTableList {
```

Comment:
Remove the extra level of indentation. Please: setup your IDE to run `go-fmt` on every save.

Reply:
I have that setup already but somehow my VS code doesn't do that automatically as I have autosave ON and `go-fmt` doesn't run on autosave I think so I had to save the file manually and then it does run it and I sometimes miss saving the file manually.